neuro_san/http_sidecar/handlers/base_request_handler.py

```diff
@@ -53,21 +54,27 @@ class BaseRequestHandler(RequestHandler):
     request_id: int = 0
 
     # pylint: disable=attribute-defined-outside-init
+    # pylint: disable=too-many-arguments
+    # pylint: disable=too-many-positional-arguments
```

Comment:
We really should raise the limbo stick here. I am tired of disabling both of these.
Also: attribute-defined-outside-init actually sounds like it's worth fixing soon. That's not a normal one.